src/vs/base/browser/ui/sash/sash.css

```diff
@@ -35,6 +35,10 @@
 
 /** Custom Mac Cursor */
 
+.monaco-workbench.mac.use-inline-toolbar > .monaco-sash.mac {
```

Comment:
We should not add workbench dependencies here, instead if we want to override something for widgets it should go into workbench.css